api/rpc/server.go

```diff
@@ -61,17 +61,15 @@ func (s *Server) verifyAuth(_ context.Context, token string) ([]auth.Permission,
 
 // RegisterService registers a service onto the RPC server. All methods on the service will then be
 // exposed over the RPC.
-func (s *Server) RegisterService(namespace string, service interface{}) {
+func (s *Server) RegisterService(namespace string, service interface{}, out interface{}, authEnabled bool) {
```

Comment:
there is an `authDisabled` on s already, can we use that and skip passing in the authEnabled? Never love a true or false tagged on the end of a method sig

Reply:
Great catch

Reply:
Also, there is not much to gain from per service auth granularity